src/Subpages/Resources/AssistanceEducation.jsx

```diff
@@ -0,0 +1,164 @@
+          Conviction history should not affect educational opportunities after
+          jail or prison, yet many college and universities will collect
+          background information and have full discretion over the admissions
+          process.
+        </Typography>
+        <Grid className={classes.regularContainer} container>
+          <Grid item xs={12} sm={6}>
+            <Typography>
+              Writing a letter of explanation, gathering refrences, and
+              providing any further aditional information may help in the
+              application process.
+            </Typography>
+          </Grid>
+          <Grid item xs={12} sm={6}>
+            <Typography>
+              You should never lie on you application. It could result in
+              voiding your application or expulsion if admitted prior to the
+              school finding out.
+            </Typography>
+          </Grid>
+        </Grid>
+      </Container>
+      <Box className={classes.lightBlueBackGround}>
+        <Container maxWidth="lg">
+          <ListItemMobileSnap
+            textLeft
+            text={
+              <div>
+                <Typography className={classes.headingStyle} variant="h2">
+                  Financial student aid
+                </Typography>
+                <Typography>
+                  For the most part, restrictions on recieving financial aid are
+                  removed once released. As long as the conditions of your
+                  sentence are being met, you will most likey be able to benefit
+                  from federal student aid, as well as a few grants and
+                  scholarships.
+                </Typography>
+              </div>
+            }
+            image={<Box component="img" src={wallet} height={300} />}
+          />
+        </Container>
+      </Box>
+      <Container>
+        <Typography className={classes.headingStyle} variant="h2">
+          Federal Student Aid
+        </Typography>
+        <Typography className={classes.contentTextStyle} variant="body1">
+          The U.S. Department of Education's office of Federal Student Aid
+          provides more than $120 billion in financial aid to help pay for
+          college or career school each year. Below are ones that apply to the
+          former incarcerated.
+          <br />
+          <br />
+          If you want to know more, check out the Federal Government’s{" "}
+          <Link
+            href="https://studentaid.ed.gov/sites/default/files/aid-info-for-incarcerated-individuals.pdf"
+            color="initial"
+          >
+            FAQ on Incarcerated Individuals and Eligibility for Federal Student
+            Aid.
+          </Link>
+        </Typography>
+        <Box className={classes.regularContainer}>
+          <IconWithHeaderAndText
+            icon={LocalAtmOutlined}
+            header={
+              <Link color="initial" href="http://www.fafsa.gov">
```

Comment:
Could be a temporary issue but this link doesn't work for me. Maybe use `https://studentaid.gov/h/apply-for-aid/fafsa`?